<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordViewWidget</class>
 <widget class="QWidget" name="RecordViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="recordViewTitle">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Light</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Description de l'enregistrement</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans Light</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Date de l'enregistrement:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="recordDate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans Light</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Position:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="imuPosition">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="19" column="0">
          <widget class="QPushButton" name="fullGraphButton">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton{background-color: rgba(119, 160, 175, 0.7);
                     border-style: inset;
                     border-width: 2px;
                     border-radius: 10px;
                     border-color: white;
                     font: 12px;
                     min-width: 10em;
                     padding: 6px; }
                     QPushButton:pressed { background-color: rgba(70, 95, 104, 0.7)}</string>
           </property>
           <property name="text">
            <string>Graphique détaillé</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans Light</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Centralle inertielle:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="imuType">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="19" column="1">
          <widget class="QPushButton" name="algoBtn">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>0</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton{background-color: rgba(119, 160, 175, 0.7);
                     border-style: inset;
                     border-width: 2px;
                     border-radius: 10px;
                     border-color: white;
                     font: 12px;
                     min-width: 10em;
                     padding: 6px; }
                     QPushButton:pressed { background-color: rgba(70, 95, 104, 0.7)}</string>
           </property>
           <property name="text">
            <string>Choisir algorithme</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="deleteRecordButton">
           <property name="minimumSize">
            <size>
             <width>156</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Supprimer l'enregistrement de la base de données</string>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton{background-color: rgba(231, 76, 60,0.7);
                     border-style: inset;
                     border-width: 2px;
                     border-radius: 10px;
                     border-color: white;
                     font: 12px;
                     min-width: 10em;
                     padding: 6px; }
                     QPushButton:pressed { background-color: rgba(70, 95, 104, 1)}</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../images.qrc">
             <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>175</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans Light</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Nom de l'enregistrement:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editButton">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="recordNameEdit"/>
           </item>
          </layout>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans Light</family>
             <pointsize>10</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Aperçu de l'enregistrement:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <layout class="QHBoxLayout" name="graphHorizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
          </layout>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>10</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Détails de l'enregistrement:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0" colspan="2">
          <widget class="QTextEdit" name="recordDetails">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Light</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Aucun détail sur cet enregistrement</string>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
